packages/shared/web/src/header/connect-community/connect-community.tsx

```diff
@@ -36,7 +38,7 @@ export function ConnectCommunity() {
         <DiscordIcon
           sx={{
             color: theme.palette.background.accentDefault,
-            fontSize: '3rem',
+            fontSize: '5rem',
```

Comment:
Can you convert all these font sizes to px and align to the typography if possible?